src/config.js

```diff
@@ -483,6 +483,23 @@ const validators = {
     default: false,
     isClient: true
   }),
+  ONBOARDING_ASSIGNMENT_ID: num({
+    desc:
+      "Assignment ID used by api/lib/twilio-incoming to onboard new contacts",
+    default: undefined
+  }),
```

Comment:
Not a suggestion, rather a question: Do you know what twilio/api/lib uses this for? 

ie. I'm not sure what ID I would put here.